brave/brave-core
# Refactor new tab page usage of RecyclerView

Resolves https://github.com/brave/brave-browser/issues/21737
Resolves https://github.com/brave/brave-browser/issues/22439
Resolves https://github.com/brave/brave-browser/issues/22434

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
### Testing orientation fix
1. Enable Brave News
2. Open a new tab page
3. Change tablet orientation to landscape
4. Begin scrolling through news feed
5. Open another new tab
6. Begin scrolling through feed in the new tab
7. Change orientation to portrait
8. Continue scrolling news
9. During each orientation change, the screen should fill the viewport (no gaps)

### Testing velocity fix
1. Launch Brave and enable the Brave News feed
2. Once enabled, start swiping through the news feed. scrolling velocity should be smooth

### Testing settings bar fix
1. Clean Install
2. Enable Brave News, enable show brave news
3. Scroll News feed
4. Observe news settings bar
